docs/app/pages/components/sections/panels/item-display-panel-inline/item-display-panel-inline.component.html

```diff
@@ -1,6 +1,6 @@
 <button class="btn button-primary m-b-sm" (click)="togglePanel()">Toggle Preview</button>
```

Comment:
Seems to be a few styling issues in plunker:

![image](https://user-images.githubusercontent.com/20795331/28166300-46a6f656-67cf-11e7-85a4-e8d6907f0162.png)